README.md

```diff
@@ -6,22 +6,21 @@ This action only runs on macOS.
 
 ## Inputs
 
-All inputs are required.
-
-| Name | Description | Example Value |
-| ---- | ----------- | ------------- |
-| certificate | The certificate for signing | `${{ secrets.CERTIFICATE }}` |
-| certificate-password | The password for the certificate | `${{ secrets.CERTIFICATE_PASSWORD }}` |
-| username | The Apple ID username to use for notarization | `${{ secrets.APPLE_ID_USERNAME }}` |
-| password | The Apple ID password to use for notarization | `${{ secrets.APPLE_ID_PASSWORD }}` |
-| apple-team-id | The Apple Team ID to use for signing and notarization | `33DS2ZRDST` |
-| app-path | The path to the application to sign and notarize | `build/my_app` |
+| Name | Description | Example Value                         | Required |
+| ---- | ----------- |---------------------------------------| -------- |
+| certificate | The certificate for signing | `${{ secrets.CERTIFICATE }}`          | Yes |
+| certificate-password | The password for the certificate | `${{ secrets.CERTIFICATE_PASSWORD }}` | Yes |
+| username | The Apple ID username to use for notarization | `${{ secrets.APPLE_ID_USERNAME }}`    | Yes |
+| password | The Apple ID password to use for notarization | `${{ secrets.APPLE_ID_PASSWORD }}`    | Yes |
+| apple-team-id | The Apple Team ID to use for signing and notarization | `33DS2ZRDST`                          | Yes |
+| app-path | The path to the application to sign and notarize | `build/my_app`                        | Yes |
+| entitlements-path | The path to the entitlements file to use for signing | `src/entitlements.plist`              | No |
```

Comment:
No need for the spaces. Either we should align it for fixed-font, or we should just not add them.
```suggestion
| certificate | The certificate for signing | `${{ secrets.CERTIFICATE }}` | Yes |
| certificate-password | The password for the certificate | `${{ secrets.CERTIFICATE_PASSWORD }}` | Yes |
| username | The Apple ID username to use for notarization | `${{ secrets.APPLE_ID_USERNAME }}` | Yes |
| password | The Apple ID password to use for notarization | `${{ secrets.APPLE_ID_PASSWORD }}` | Yes |
| apple-team-id | The Apple Team ID to use for signing and notarization | `33DS2ZRDST` | Yes |
| app-path | The path to the application to sign and notarize | `build/my_app` | Yes |
| entitlements-path | The path to the entitlements file to use for signing | `src/entitlements.plist` | No |
```